upgrade/1.2/scripts/upgrade/csm-upgrade.sh

```diff
@@ -121,5 +121,6 @@ else
     echo "====> ${state_name} has been completed"
 fi
 
+ GOSS_BASE=/opt/cray/tests/install/ncn goss -g /opt/cray/tests/install/ncn/suites/ncn-upgrade-preflight-tests.yaml --vars=/opt/cray/tests/install/ncn/vars/variables-ncn.yaml validate
```

Comment:
I think it would be good to add an "echo" before this indicating that it is running a set of tests.  That would better explain the output that they will see when the tests run.

```
....................

Total Duration: 7.117s
Count: 20, Failed: 0, Skipped: 0
```